config/config.go

```diff
@@ -188,6 +188,9 @@ type AgentConfig struct {
 	// If set to -1, no archives are removed.
 	LogfileRotationMaxArchives int `toml:"logfile_rotation_max_archives"`
 
+	// Inidicates whether or not to use local time as the logging prefix.
+	LogWithLocalTime bool `toml:"log_with_local_time"`
```

Comment:
something like `log_with_timezone` where timezone is a string and can be `local` or any other timezone would be quite useful and a bit more flexible.

Reply:
Good idea, done!